netbench/cdk/lambda/timestamp.js

```diff
@@ -0,0 +1,9 @@
+exports.handler =  async (event, context) => {
+    var current = new Date();
+    var time = current.toLocaleDateString() + "-" + current.getHours() + ":" + current.getMinutes() + ":"
+        + current.getSeconds() + ":" + current.getMilliseconds();
```

Comment:
Do you need the date and time in this format, or could you just use millis since the epoch?

Reply:
Since I'm using this timestamp as the name for directories in S3 to identify runs, I figured a more human readable timestamp would be better.